lib/input/azure_queue_storage_config.go

```diff
@@ -51,8 +51,8 @@ Only one authentication method is required, ` + "`storage_connection_string`" +
 				"A storage account connection string. This field is required if `storage_account` and `storage_access_key` / `storage_sas_token` are not set.",
 			),
 			docs.FieldCommon(
-				"queue_name", "The name of the target Storage queue.",
-			),
+				"queue_name", "The name of the source Storage queue.",
+			).IsInterpolated(),
```

Comment:
Might be worth adding an example here on how this might be used as it might be confusing given the absense of an input message.

Reply:
Added example

Reply:
Yes, I agree it's confusing because it'll never be interpolated from the `msg` since it was not yet produced, but this is actually a real use case where we want the queue name based on an env var, which value comes in PascalCase and Azure Storage Queue only supports queue names in lowercase.